finality-aleph/src/lib.rs

```diff
@@ -180,5 +191,8 @@ where
     C::Api: aleph_primitives::AlephSessionApi<B>,
     SC: SelectChain<B> + 'static,
 {
-    run_consensus_party(AlephParams { config })
+    match config.node_type {
+        NodeType::Validator => run_validator_node(AlephParams { config }).boxed(),
```

Comment:
This if intersport pattern doesn't look good here ;) Perhaps if there would be anything beyond just forwarding that call. That match in main should be more than enough and you already divided it, i.e. `new_full` and `new_nonvalidator`.

Reply:
without the node_type this is no longer here :)